inference-engine/samples/benchmark_app/remote_helper.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#ifdef USE_REMOTE_MEM
+#include <string>
+#include <algorithm>
+#include <utility>
+#include <vector>
+#include <map>
+#include <regex>
+
+#include <inference_engine.hpp>
+
+#include <samples/common.hpp>
+#include <samples/slog.hpp>
+
+#include "utils.hpp"
+#include "remote_helper.hpp"
+#include "WorkloadContext.h"
+#include "RemoteMemory.h"
+#include "hddl2/hddl2_params.hpp"
+#include "ie_compound_blob.h"
+
+using namespace InferenceEngine;
+
+class RemoteHelper::Impl {
```

Comment:
Is it better if name it RemoteContextHelper?

Reply:
Done.